docs/source/install.rst

```diff
@@ -11,30 +11,35 @@ Here are two ways to hack on python.org:
 Easy setup using Vagrant
 ------------------------
 
+First, install install Vagrant_ and Ansible_ on your machine. 
+You should then be able to provision the Vagrant box
+
 ::
 
     $ vagrant up
-    $ vagrant ssh
-    # go to pythondotorg/ directory and activate virtualenv, then run
-    $ ./manage.py runserver 0.0.0.0:8000
-    # on your local shell
-    $ google-chrome http://localhost:8001/
+    
+The box will be provisioned with Python 3.5, a virtualenv with all
```

Comment:
Nit: virtualenv -> virtual environment (I forgot to do this when I updated the documentation a while ago :))